src/main/java/eu/bbmri_eric/negotiator/dto/discoveryservice/GenericNetwork.java

```diff
@@ -0,0 +1,20 @@
+package eu.bbmri_eric.negotiator.dto.discoveryservice;
+
+public interface GenericNetwork {
```

Comment:
please add javadoc

Reply:
Fixed in 3235f439a3ce3690a16b4781597181da46cebe37